### PR TITLE
add swapaxes

### PR DESCRIPTION
**Pull request by mtar**
## Description

Adds swapaxes. It is mostly an alias function for transpose on two axes.

Issue/s resolved: #849 

## Changes proposed:
- add swapaxes

## Due Diligence

- [x] All split configurations tested
- [x] Multiple dtypes tested in relevant functions
- [x] Documentation updated (if needed)
- [x] Updated changelog.md under the title "Pending Additions"

#### Does this change modify the behaviour of other functions? If so, which?
no

### CODE REVIEW

**Comment by mtar** [comment]
run tests

**Review by ClaudiaComito — CHANGES_REQUESTED** [review]
Great @mtar! Just a couple small things (removal of instance checks and implementation of method)

File: heat/core/tests/test_manipulations.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from typing import Type
```
[the remark this answers is not in the document]
Comment (mtar) [reply heat/core/tests/test_manipulations.py]: No

File: heat/core/manipulations.py [hunk 2]
```diff
@@ -2957,6 +2958,51 @@ def stack(
...
+                type(axis1), type(axis2)
+            )
+        )
+
+    axes = list(range(x.ndim))
+    axes[axis1], axes[axis2] = axes[axis2], axes[axis1]
+
+    return linalg.transpose(x, axes)
+
+
```
[the remark this answers is not in the document]
Comment (mtar) [reply heat/core/manipulations.py:3005]: nice catch

File: heat/core/manipulations.py [hunk 3]
```diff
@@ -2957,6 +2958,51 @@ def stack(
...
+    """
+    if not isinstance(axis1, int) or not isinstance(axis2, int):
+        raise TypeError(
+            "'axis1' and 'axis2' must be of type int, found {} and {}".format(
+                type(axis1), type(axis2)
+            )
+        )
+
+    axes = list(range(x.ndim))
+    axes[axis1], axes[axis2] = axes[axis2], axes[axis1]
```
[the remark this answers is not in the document]
Comment (mtar) [reply heat/core/manipulations.py]: changed

**Review by coquelin77 — DISMISSED** [review]
same as #854 looks good with changes already addressed. merging

**Pull request merged**